docs/maintainers/readme.md

```diff
@@ -0,0 +1,48 @@
+# Maintainer Processes
+
+This document contains basic instructions for the maintainers of the DEV
+application. It's a work in-progress, but should allow the DEV Core Team to
+document processes and strategies in a transparent way!
+
+# Skipping CI
+
+Some small PRs might seem like good candidates for skipping the CI suite. For
+example, a change that only touches documentation copy or a tiny CSS tweak.
+
+For the time being, we should avoid skipping CI in PRs because it causes some
+confusion during the review process. However, it does make sense to skip CI on
+merge commits that include extremely minor changes.
+
+You can skip CI by appending [ci skip] to merge commit's message.
+
+When in doubt, don't skip CI.
+
+# Pull Request Review
+
+At the moment, our PR review process is being evaluated, so this is subject to
+change.
+
+Before a PR is merged we expect:
+
+- every contributor to sign our CLA
+- all of the automated checks to pass
+- approval from at least two members of the Core Team
+
+PR review is the Core Team's opportunity to weigh-in on changes before they go
+into master.
+
+Keep in mind that our team is distributed across the world. It's a good idea to
+leave far-reaching PRs open for a day, to give everyone a change to share their
```

Comment:
We have an SLA of 3 business days currently, as communicated by our template response:

> Thank you so much for your PR! We'll do our best to review your submission as soon as possible.
> If you don't hear from us within 3 business days, please ping @maestromac or @rhymes or @citizen428.
> Some things we look for immediately when reviewing a PR are: tests, updated docs, etc.
> Please make sure you've added all relevant info in order to help us get your code merged as quickly as possible!

Maybe we should align this, saying "a day" here and "3 business days" in the canned response could be confusing.

Reply:
Given our teams might rotate and other people might make up the OSS team in the future it might be a good idea to have them mention the team instead, for example, reach out to @thepracticaldev/oss

Reply:
Definitely @mstruve, the templates haven't been finalized yet, this is what we're using right now.